src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends Exception {
+    public AuthenticationException(String message) {
+        super(message);
+    }
+}
```

Comment:
you should add also a constructor with message and throwable to both your custom exceptions.